notebooks/demo_individual/demo_generate_climos.ipynb

```diff
@@ -0,0 +1,302 @@
+{
```

Comment:
The entire `demo_individual` directory has been removed from `master`.  We don't want to reintroduce it.